utility/helpers.js

```diff
@@ -33,6 +33,14 @@ log4js.configure({
 let logger = log4js.getLogger();
 logger.level = "debug";
 
+const Sentry = require("@sentry/node");
+Sentry.init({
+	dsn: process.env.SENTRY_DSN,
```

Comment:
[BETTER OPTION]: export SENTRY_DSN: process.env.SENTRY_DSN in config/index and use it throughout

Reply:
merging, please update in next commit

Reply:
okay will change this!